app/src/main/java/com/example/playlistmarket/data/converters/MediaLibraryDatabaseConverter.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.playlistmarket.data.converters
+
+import com.example.playlistmarket.data.db.entity.TrackEntity
+import com.example.playlistmarket.domain.search.models.Track
+
+class MediaLibraryDatabaseConverter {
+    fun map(trackEntity: TrackEntity): Track {
+        return Track(
+            trackEntity.previewUrl,
+            trackEntity.trackName,
+            trackEntity.artistName,
+            trackEntity.trackTimeMillis,
+            trackEntity.artworkUrl100, trackEntity.trackId,
+            trackEntity.collectionName,
+            trackEntity.releaseDate,
+            trackEntity.primaryGenreName,
+            trackEntity.country)
+    }
```

Comment:
Подобные функции можно писать проще и лаконично
```suggestion
    fun map(trackEntity: TrackEntity): Track = Track(
            trackEntity.previewUrl,
            trackEntity.trackName,
            trackEntity.artistName,
            trackEntity.trackTimeMillis,
            trackEntity.artworkUrl100, trackEntity.trackId,
            trackEntity.collectionName,
            trackEntity.releaseDate,
            trackEntity.primaryGenreName,
            trackEntity.country)
```